From 65f0a251c081efa3cca725621f5a5c2ce545fdbf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Thu, 3 Sep 2015 15:31:55 +0200
Subject: [PATCH] Adapt to ExtUtils-MakeMaker-6.07
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

ExtUtils-MakeMaker-6.07 changes format of generated Makefile,
therefore a regular expression in t/Util.pm could not locate
interested line and t/default_make_test/with_extra_tests.t test
failed.

This patch augemnts the regular expression to work with
ExtUtils-MakeMaker-6.07 either.

CPAN RT#106843

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 t/Util.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/Util.pm b/t/Util.pm
index 2757f4f..70b5904 100644
--- a/t/Util.pm
+++ b/t/Util.pm
@@ -38,7 +38,7 @@ sub find_make_test_command {
         open my $fh, '<', 'Makefile' or die "Cannot open 'Makefile' for reading: $!";
         my $regex = _regex(keys %$target);
         while (<$fh>) {
-            next unless /^ ($regex) \s+ :: \s+ (?:pure_all|$regex) /xms;
+            next unless /^ ($regex) \s+ :: \s+ (?:pure_all|$regex|subdirs-(?:$regex)) /xms;
             $commands->{$1} = scalar <$fh>;
             delete $target->{$1};
             my @target = keys %$target;
-- 
2.4.3

